### ISSUE TITLE
Display Data from the Northwind Service

### ISSUE THREAD

**Issue by sandraros** [SAPDocuments/Issues#582]
Tutorials: https://developers.sap.com/tutorials/sapui5-fiori-cf-display-data.html
--------------------------

Could you indicate that the spaces are very important in the YAML file. It's not easy to understand for learners/people who don't know YAML.
We blindly copy/paste the values, so maybe replacing that:
```
destinations:
  - name: "Northwind"
    url: "https://services.odata.org/"
```
with that:
```
        destinations:
          - name: "Northwind"
            url: "https://services.odata.org/"
```
will help people (maybe).
Thanks! :)

**Comment by IObert** [comment]
I agree, it can be very annoying if you don't know this. I added a note that should be visible in the next few minutes.

Thanks for your feedback!
